openwisp_monitoring/device/migrations/0004_wificlient_wifisession.py

```diff
@@ -0,0 +1,141 @@
+                ('vht', models.BooleanField(default=False, verbose_name='VHT')),
+                ('wmm', models.BooleanField(default=False, verbose_name='WMM')),
+                ('wds', models.BooleanField(default=False, verbose_name='WDS')),
+                ('wps', models.BooleanField(default=False, verbose_name='WPS')),
+            ],
+            options={
+                'verbose_name': 'WiFi Client',
+                'abstract': False,
+                'swappable': 'DEVICE_MONITORING_WIFICLIENT_MODEL',
+            },
+        ),
+        migrations.CreateModel(
+            name='WifiSession',
+            fields=[
+                (
+                    'id',
+                    models.UUIDField(
+                        default=uuid.uuid4,
+                        editable=False,
+                        primary_key=True,
+                        serialize=False,
+                    ),
+                ),
+                (
+                    'modified',
+                    model_utils.fields.AutoLastModifiedField(
+                        default=django.utils.timezone.now,
+                        editable=False,
+                        verbose_name='modified',
+                    ),
+                ),
+                (
+                    'ssid',
+                    models.CharField(
+                        blank=True, max_length=32, null=True, verbose_name='SSID'
+                    ),
+                ),
+                ('interface_name', models.CharField(max_length=15)),
+                (
+                    'start_time',
+                    models.DateTimeField(
+                        auto_now=True, db_index=True, verbose_name='start time'
+                    ),
+                ),
+                (
+                    'stop_time',
+                    models.DateTimeField(
+                        blank=True, db_index=True, null=True, verbose_name='stop time'
+                    ),
+                ),
+                (
+                    'device',
+                    models.ForeignKey(
+                        on_delete=django.db.models.deletion.CASCADE,
+                        to=swapper.get_model_name('config', 'Device'),
+                    ),
+                ),
+                (
+                    'wifi_client',
+                    models.ForeignKey(
+                        on_delete=django.db.models.deletion.CASCADE,
+                        to=swapper.get_model_name('device_monitoring', 'WifiClient'),
+                    ),
+                ),
+            ],
+            options={
+                'verbose_name': 'WiFi Session',
+                'ordering': ('-start_time',),
+                'abstract': False,
+                'swappable': 'DEVICE_MONITORING_WIFISESSION_MODEL',
+            },
+        ),
+        migrations.RunPython(
+            assign_permissions_to_groups, reverse_code=migrations.RunPython.noop
```

Comment:
please run this migration in a separate data migration step (file), do you remember on postgres we get errors when we try to do changes to the data in the same transaction in which the schema is changed? Please don't forget.. we already ran into this issue.